docs/sql-ref-datetime-pattern.md

```diff
@@ -58,8 +58,8 @@ Spark uses pattern letters in the following table for date and timestamp parsing
 |**Z**|zone-offset|offset-Z|+0000; -0800; -08:00;|
 |**'**|escape for text|delimiter||
 |**''**|single quote|literal|'|
-|**[**|optional section start|||
-|**]**|optional section end|||
+|**[**|optional section start| | |
```

Comment:
Need the space between `|`s? But, no space between `|`s in line 59?

Reply:
it would be nicer if we have an extra space between | and |

Yes. Line 59 is needed too

Reply:
@maropu @gatorsmile OK.